styles/AuthModal.module.css

```diff
@@ -1,10 +1,15 @@
 .modalWrapper {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
   width: 325px;
-  height: 525px;
+  height: 535px;
```

Comment:
Consider ultrawide or super high resolution monitors here. Last time I worked extensively with styling, I custom built all of it which was such a pain in the butt. There m ight be helpful libraries out there to help design responsively

Reply:
Yeah, some of the styling I've been doing is just to give my brain a break from the TypeScript/Mapbox work. I definitely do want to find some kind of out-of-the-box solution to this, I have a super ultrawide myself and getting anything to display properly on it takes a lot of work. I can only imagine how wonky it'll look on something like a 4k or 4k ultrawide.